mws/apis/inbound_shipments.py

```diff
@@ -429,7 +429,20 @@ def put_transport_content(self, shipment_id, is_partnered, shipment_type, **kwar
         else:
             data = self._transport_details_non_partnered(shipment_type, data, **kwargs)
 
-        return self.make_request(data, method="POST", use_body_params=True)
+        try:
+            return self.make_request(data, method="POST", use_body_params=True)
+        except MWSError as e:
+            try:
+                error = DictWrapper(e.response.content).parsed.get("Error", {})
```

Comment:
Don't we need to include this requirement in the project and import it in order to use it?